tests/ecomm.tests.ts

```diff
@@ -6,18 +6,19 @@ describe('Ecomm tests', () => {
   let duda: Duda;
   let scope: nock.Scope;
 
-  const site_name = "test_site";
-  const product_id = "test_product";
-  const gateway_id = "test_gateway";
-  const cart_id = 'test_cart';
-  const order_id = 'test_order';
-  const refund_id = 'test_refund';
-  const session_id = 'test_session';
-  const category_id = 'test_category';
-  const shipping_id = 'test_shipping';
-  const option_id = 'test_option';
+  const site_name = 'string';
```

Comment:
Is there a reason you changed these to all be the same? I'm just afraid it could be easy to have a test pass when it shouldn't by doing something like `product_id === 'string'` when you mean `gateway_id === 'string'` having different values might be better.